aims/aim3.md

```diff
@@ -0,0 +1,60 @@
+These manifolds constrain datasets to occupy a small subset of total space and can contain vital information regarding data generating functions.
+Remarkably, certain classes of generative models, including variational autoencoders (VAE) and generative adversarial networks (GANs), that add a distribution matching constraint, imbue intuitive mathematical features to the learned latent features [1,2].
+For instance, a recent GAN learned latent representations that could be manipulated with arithmetic: Subtracting out the essence of a smile from a woman and adding it to a neutral man resulted in an image vector of a smiling man [3].
+While image data have been extensively modeled, less is known about the learning capacity of such models applied to single cell gene expression data.
+The _objective of this aim_ is to assess the extent to which single cell gene expression latent spaces can be manipulated and how such manifold preserving properties will enable biological discovery regarding cell type, cellular differentiation, and disease perturbation.
+The _working hypothesis of this aim_ is that the latent space will preserve arithmetic operations between state transitions of differentiating hematopoietic cells.
+
+## Preliminary Data:
+
+We have previously modeled bulk gene expression data with denoising autoencoders and non-linear activation functions [4–6].
+These models have engineered features that encode information about gene ontology terms and KEGG pathways.
+We have shown that these models capture more information than linear models like PCA, ICA, and NMF, and that the learned features are often overlapping and redundant [7].
+We have also recently trained a VAE on cancer gene expression datasets and observed that the learned latent features capture interesting biological distributions of features across different tumors [8].
+Therefore, we are confident that modeling gene expression with reconstruction cost captures meaningful biology.
+Furthermore, we trained VAEs and GANs on electronic health record (EHR) data including a model to generate privacy aware fake datasets to supplement learning or data sharing [9,10].
+We will develop similar approaches to extract knowledge from Human Cell Atlas (HCA) single cell data.
+
+## Enabling latent space arithmetic in single cell gene expression with variational autoencoders:
+
+Once a VAE is trained on HCA single cell gene expression data, we will obtain the latent space representation of a heterogeneous population of blood cells.
+We will use well characterized gene expression markers and unsupervised clustering algorithms to define core subsets of common blood cell types.
+We will define core subsets by positive silhouette widths of well characterized cell types representing a variety of the differentiation trajectories of hematopoiesis.
+We will test our hypothesis that the learned latent space vectors can be manipulated mathematically to obtain biological insights and useful properties.
```

Comment:
yes, will be updated for pertrub-Seq experiment.

Current thinking: Identify latent vector of perturbation response (cell A - cell A w/ pert) and ask if (cell B + cell A w/ pert) = cell B w/ pert.

Another way to do this would be to identify all "cell types" perturbation latent vector and ask if they are closer to each other in latent euclidean space than in raw euclidean space. We can also identify the "essence" of the latent vector (w/ perturb) to see if its in line with the known downstream consequences of fibroblast --> cardiomyocyte TFs.